src/styles/typo.ts

```diff
@@ -0,0 +1,27 @@
+import { css } from '@emotion/react';
+
+const generateTypo = (size: number, weight: number) => css`
+  font-family: 'Pretendard';
```

Comment:
fallback font도 같이 설정해주면 어떨까?
```suggestion
  font-family: "Pretendard Variable", Pretendard, -apple-system, BlinkMacSystemFont, system-ui, Roboto, "Helvetica Neue", "Segoe UI", "Apple SD Gothic Neo", "Noto Sans KR", "Malgun Gothic", "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", sans-serif;

```